genesyscloud/responsemanagement_responseasset/genesyscloud_responsemanagement_responseasset_proxy.go

```diff
@@ -0,0 +1,110 @@
+	return p.createRespManagementRespAssetAttr(ctx, p, respAsset)
+}
+
+// updateRespManagementRespAsset updates a Genesys Cloud responsemanagement responseasset by Id
+func (p *responsemanagementResponseassetProxy) updateRespManagementRespAsset(ctx context.Context, id string, respAsset *platformclientv2.Responseassetrequest) (*platformclientv2.Responseasset, int, error) {
+	return p.updateRespManagementRespAssetAttr(ctx, p, id, respAsset)
+}
+
+// getRespManagementRespAssetById returns a single Genesys Cloud responsemanagement responseasset by Id
+func (p *responsemanagementResponseassetProxy) getRespManagementRespAssetById(ctx context.Context, id string) (*platformclientv2.Responseasset, *platformclientv2.APIResponse, error) {
+	return p.getRespManagementRespAssetByIdAttr(ctx, p, id)
+}
+
+// deleteRespManagementRespAsset deletes a Genesys Cloud responsemanagement responseasset by Id
+func (p *responsemanagementResponseassetProxy) deleteRespManagementRespAsset(ctx context.Context, id string) (response *platformclientv2.APIResponse, err error) {
+	return p.deleteRespManagementRespAssetAttr(ctx, p, id)
+}
+
+// createRespManagementRespAssetFn is an implementation of the function to create a Genesys Cloud responsemanagement responseasset
+func createRespManagementRespAssetFn(ctx context.Context, p *responsemanagementResponseassetProxy, respAsset *platformclientv2.Createresponseassetrequest) (*platformclientv2.Createresponseassetresponse, int, error) {
+	postResponseData, resp, err := p.responseManagementApi.PostResponsemanagementResponseassetsUploads(*respAsset)
+	if err != nil {
+		return nil, 0, fmt.Errorf("Failed to upload response asset: %v", err)
+	}
+	return postResponseData, resp.StatusCode, nil
+}
+
+// updateRespManagementRespAssetFn is an implementation of the function to update a Genesys Cloud responsemanagement responseasset
+func updateRespManagementRespAssetFn(ctx context.Context, p *responsemanagementResponseassetProxy, id string, respAsset *platformclientv2.Responseassetrequest) (*platformclientv2.Responseasset, int, error) {
+	putResponseData, resp, err := p.responseManagementApi.PutResponsemanagementResponseasset(id, *respAsset)
+	if err != nil {
+		return nil, 0, fmt.Errorf("Failed to update Responsemanagement response asset %s: %v", id, err)
+	}
+	return putResponseData, resp.StatusCode, nil
+}
+
+// getRespManagementRespAssetByIdFn is an implementation of the function to get a Genesys Cloud responsemanagement responseasset by Id
+func getRespManagementRespAssetByIdFn(ctx context.Context, p *responsemanagementResponseassetProxy, id string) (*platformclientv2.Responseasset, *platformclientv2.APIResponse, error) {
+	sdkAsset, resp, getErr := p.responseManagementApi.GetResponsemanagementResponseasset(id)
+	if getErr != nil {
+		return nil, nil, fmt.Errorf("failed to retrieve response asset: %s", getErr)
```

Comment:
Same here, return the resp when there's an error. Same for the update and delete